src/content/docs/en/installation/rpi.mdx

```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
   it on another device.
 </Alert>
 
+First, you will need to decompress it.
+
+- On Linux, run `unzstd *FILENAME.img.zst*`, you may need to install the `zstd` package
```

Comment:
unzstd isn’t included by default, maybe suggest the zstd -d method

Reply:
It's a part of zstd